### PR TITLE
Introduce global configuration file

### PR DESCRIPTION
**Pull request by Surzhko**
This changes allows to extract main configuration data (all except `user_and_repo`) to global config to home directory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+{
+    "oauth_token":              "TOKEN",
+    "user_and_repo":            "ORG/REPO",
+    "trello_key":               "KEY",
+    "trello_token":             "TOKEN",
+    "trello_board_id":          "BOARD_ID",
+    "trello_platform_board_id": "BOARD_ID"
+}
+```
+But all general information except `user_and_repo` could be extracted to global config to home directory with the same name
```
Comment (arutinn) [inline README.md:14]: :+1:

**Review by BrianBorge — APPROVED** [review]
Global config! 🥳 

Thanks for adding this.  I think the next iteration on this should try to remove the requirement from having a config file per project with`user_name_and_repo` entirely.

The reason for this: If I want to make use of the global config, I am required to have at least two config files, one in the user dir and one in the project I am working on.  It would be nice to have one config file, where the project-specific config file acts as an override but is not necessary to include at all.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
+# Configuration
+
+Each repository should contains it's own config (`.github_workflow`), that should looks like:
```
Comment (BrianBorge) [inline README.md:3]: suggestion (blocking): 
```suggestion
Each repository should contain its own config (`.github_workflow`), which should look like this:
```

File: README.md (hunk 1 above)
Comment (BrianBorge) [inline README.md:14]: suggestion (non-blocking):

```suggestion
Optionally, all general information except `user_and_repo` could be extracted to global config in your home directory with the same name (~./github_workflow)
```

File: lib/github_workflow/cli.rb [hunk 3]
```diff
@@ -11,6 +11,7 @@ module GithubWorkflow
   class Cli < Thor
     PROCEED_TEXT = "Proceed? [y,yes]: ".freeze
 
+    CONFIG_FILE_NAME =".github_workflow".freeze
```
Comment (BrianBorge) [inline lib/github_workflow/cli.rb:14]: suggestion(blocking): Add space after constant and space after `=`

```suggestion
    CONFIG_FILE_NAME = ".github_workflow".freeze

```

**Review by BrianBorge — CHANGES_REQUESTED** [review]

File: lib/github_workflow/cli.rb (hunk 3 above)
Comment (Surzhko) [reply lib/github_workflow/cli.rb:14]: :facepalm:

**Comment by Surzhko** [comment]
Yes, I have more ideas how to improve configuration, and go out only to global config with possibility override some project specific options

**Review by BrianBorge — APPROVED** [review]
Looks good.

**Pull request merged**